apps/ledger-live-mobile/src/components/RootNavigator/MarketNavigator.tsx

```diff
@@ -19,19 +18,16 @@ export default function MarketNavigator() {
         title: "",
         headerRight: undefined,
         headerLeft: () => null,
+        headerTransparent: true,
       }
     : {
         headerShown: false,
+        headerTransparent: true,
       };
 
   return (
     <Stack.Navigator screenOptions={stackNavigationConfig} initialRouteName={ScreenName.MarketList}>
       <Stack.Screen name={ScreenName.MarketList} component={MarketList} options={headerConfig} />
-      <Stack.Screen
-        name={ScreenName.MarketDetail}
-        component={MarketDetail}
-        options={headerConfig}
-      />
```

Comment:
Not used?

Reply:
Already defined in the BaseNavigator. Today when you navigate from the market to MarketDetail you keep the tabBar. When you navigate from an asset to the MarketDetail you don't have it. I discussed it with Cédric and we decided to remove the TabBar when you're on MarketDetail. So I removed this Screen definition to only keep the one in BaseNavigator.

Reply:
Oke nice ! :)